website/views/shared/header/menu.jade

```diff
@@ -99,14 +99,14 @@ nav.toolbar(ng-controller='MenuCtrl')
               a(ui-sref='options.profile.achievements')=env.t('achievs')
             li
               a(ui-sref='options.profile.profile')=env.t('profile')
-      li.toolbar-button-dropdown
+      li.toolbar-button-dropdown(dropdown)
         a(ui-sref='options.social.inbox', ng-if='user.inbox.newMessages')
           span.badge.badge-danger {{user.inbox.newMessages}}
-        a(ui-sref='options.social.tavern', data-close-menu)
+        a(ui-sref='options.social.tavern')
           span=env.t('social')
-        a(ng-class='{active: _expandedMenu.menu === "social"}', data-expand-menu, menu='social')
+        a(dropdown-toggle)
           span &#9776;
-        div(ng-show='_expandedMenu.menu === "social"', data-close-menu)
```

Comment:
Not sure, it seems they always have the inactive class right now but that doesn't change anything in the CSS